build/deploy/Dockerfile

```diff
@@ -3,9 +3,9 @@ FROM debian:jessie
 MAINTAINER Tobias Schottdorf <tobias.schottdorf@gmail.com>
 
 RUN mkdir -p /cockroach
-COPY cockroach.sh test.sh build/cockroach /cockroach/
 COPY build/resources /cockroach/resources
 COPY build/ui /cockroach/ui
+COPY cockroach.sh test.sh build/cockroach /cockroach/
```

Comment:
Eh? The docs are slightly unclear here. I would have thought `COPY <srcdir1> <srcdir2> <destdir>` would create `<destdir>/$(basename <srcdir1>)` and `<destdir>/$(basename <srcdir2>)` like `cp -r` would.